app/models/create_cap_proposal.rb

```diff
@@ -0,0 +1,31 @@
+class CreateCapProposal
+  def initialize(auction)
+    @host = ENV.fetch('C2_HOST', 'https://cap.18f.gov')
+    @c2_client = C2::Client.new(
+      oauth_key: ENV.fetch('C2_OAUTH_KEY'),
+      oauth_secret: ENV.fetch('C2_OAUTH_SECRET'),
+      host: @host,
+      debug: ENV.fetch('C2_DEBUG', false)
+    )
+  end
+
+  def perform
+    resp = c2_client.post 'proposals', {
+      gsa18f_procurement: {
+        product_name_and_description: "some stuff",
+        cost_per_unit: 123.00,
+        quantity: 1,
+        justification: "because because because",
+        link_to_product: "18f.gov",
+        purchase_type: "Software"
+      }
+    }
+    proposal = resp.body
```

Comment:
ah ok so the response includes the `id` of the proposal? that's cool. we might consider asking them to return the full URL as well since that would be in line with HATEOAS https://en.wikipedia.org/wiki/HATEOAS